data/mp/stats/weapons.json

```diff
@@ -4352,7 +4352,7 @@
 		"movement": "HOMING-DIRECT",
 		"muzzleGfx": "FXATMiss.PIE",
 		"name": "VTOL Sunburst AA",
-		"numAttackRuns": 6,
+		"numAttackRuns": 2,
```

Comment:
Too much.
A threefold reduction is not justified.

Reply:
With such an ammunition of 6 Attack Runs of 6 missiles each, it allows a small number of these planes to destroy three times as many other planes without having to retreat to the VTOL Pad to reload. I think this is an imbalance, especially since the VTOL Sunburst has previously been improved in accuracy and reduced time between Attack Runs (compare with version 3.4.1)

Reply:
I am not against weakening, but I think three-fold change is overkill. Better to make two changes a little at a time. Too big changes at one time often lead to problems (remember what happened to the bullet.)

Reply:
@vaut Have someone host a game with this `master balance + this PR` mod and find out. I set the `numAttackRuns` value to 3 in this mod though you can change it to whatever else you want.

[masterBalance.zip](https://github.com/Warzone2100/warzone2100/files/7218221/masterBalance.zip)

Reply:
One plane with numAttackRuns 2 is enough to kill cobra+heap bombs, but not enough against panther+heap bombs. I think numAttackRuns 3 will be enough